index.html

```diff
@@ -60,108 +67,133 @@ <h1 id="nm">I’m Mariane<br><br>
         </section>
 
         <section class="works">
-            <div class="card">
+            <div class="card order3">
                 <div>
                     <img src="./assets/Snapshoot Portfolio.png" alt="">
-                    <h2>Tonic</h2>
-                    <div class="details">
-                        <h3><b>CANOPY</b></h3>
-                        <ul>
-                            <li>&bull;&nbsp;&nbsp; Back End Dev</li>
-                            <li>&bull;&nbsp;&nbsp; 2015</li>
+                    <div class="card-wrap">
+                        <h2>Tonic</h2>
+                        <div class="details">
+                            <h3><b>FACEBOOK</b></h3>
+                            <ul>
+                                <li>&bull;&nbsp;&nbsp; Full Stack Dev</li>
+                                <li>&bull;&nbsp;&nbsp; 2015</li>
+                            </ul>
+                        </div>
+                        <p>A daily selection of privately<br class="break"> personalized reads; no accounts or<br
+                                class="break"> sign-ups required.
+                        </p>
+                        <ul class="tech">
+                            <li>html</li>
+                            <li>css</li>
+                            <li class="js">javascript</li>
                         </ul>
+                        <a href="#"><button type="reset">See Project</button></a>
```

Comment:
- [ ] I noticed that the text of this button element is not aligned correctly @ exactly `min-width: 786px`, kindly resolve this typo for all the similar elements as well. Thanks. 👍